src/Microsoft.DocAsCode.MarkdownLite/JsonTokenTreeRenderer.cs

```diff
@@ -274,6 +274,22 @@ protected string Escape(string content)
                 .Replace("'", "&#39;");
         }
 
+        protected string ExposeTokenName(IMarkdownToken token)
+        {
+            var tokenName = token.GetType().Name;
+            if (tokenName.StartsWith("Markdown"))
+                tokenName = tokenName.Substring("Markdown".Length, tokenName.Length - "Markdown".Length);
```

Comment:
>Substring("Markdown".Length, tokenName.Length - "Markdown".Length); [](start = 38, length = 67)

extract a method to pass startswith/endswith as parameter, avoid copying the string 3 times